conf/supervisord-dev.conf

```diff
@@ -0,0 +1,51 @@
+[supervisord]
+nodaemon = true
+silent = true
+
+[program:web]
+command = pserve --reload conf/development-app.ini
```

Comment:
We should get rid of `pserve` and run `gunicorn` directly. For consistency with the other projects that no longer use `pserve` if nothing else. But it may also make things faster and more reliable. I want to leave that to a separate PR though because it affects prod not just dev, and removing `pserve` isn't as easy in h as in other projects. For example h already contains a [giant `gunicorn.conf.py` file](https://github.com/hypothesis/h/blob/master/gunicorn.conf.py) that's used in prod

I did remove the `newrelic-admin run-program ...` prefix from the `pserve` commands though. It's used in prod, but I don't think we need to wrap everything in `newrelic` in dev.